swift-ticketing/src/swift_ticketing/worker.clj

```diff
@@ -121,21 +121,28 @@
       (ticket/cancel-tickets tx selected-ticket-ids)
       (booking/update-booking-status tx booking-id booking/CANCELED))))
 
-(defn process-ticket-requests [worker-id message-queue db-spec redis-opts]
-  (async/go
-    (while true
-      (try
-        (let [request (async/<! message-queue)
-              event-type (:event request)]
-          (println "Got Message:")
-          (println request)
-          (cond
-            (= event-type reserve-event) (handle-reserve-event db-spec redis-opts request)
-            (= event-type book-event) (handle-book-event db-spec request)
-            (= event-type cancel-event) (handle-cancel-event db-spec request)
-            :else (println "Worker: Unknown event")))
-        (catch Exception e
-          (println (str "Exception in Worker: " worker-id " :" e)))))))
+(defn process-ticket-requests [worker-id message-queue db-spec redis-opts exit-ch]
+  (async/go-loop []
```

Comment:
this function is too big and indented. Break it down into smaller chunks of work.